extensions/mysql/index.js

```diff
@@ -53,7 +51,8 @@ class MySQLExtension extends cli.Extension {
 
         return Promise.fromCallback(cb => this.connection.connect(cb)).catch((error) => {
             if (error.code === 'ECONNREFUSED') {
-                return Promise.reject(new cli.errors.ConfigError({
+                return Promise.reject(new ConfigError({
+
```

Comment:
Not really pertinent, but is this extra line needed?

Reply:
Not sure how that got added 🙃